tests/test_template.py

```diff
@@ -256,11 +244,33 @@ def test_body_with_python_template_with_sgt(self):
             os.getcwd(),
             "tests/fixtures/templates/vpc_sgt.py"
         )
-        actual_output = json.loads(self.template.body)
+        actual_output = yaml.safe_load(self.template.body)
         with open("tests/fixtures/templates/compiled_vpc.json", "r") as f:
             expected_output = json.loads(f.read())
         assert actual_output == expected_output
 
+    def test_body_injects_yaml_start_marker(self):
+        self.template.name = "vpc"
+        self.template.path = os.path.join(
+            os.getcwd(),
+            "tests/fixtures/templates/vpc.without_start_marker.yaml"
+        )
+        output = self.template.body
+        with open("tests/fixtures/templates/vpc.yaml", "r") as f:
+            expected_output = f.read()
+        assert output == expected_output
+
+    def test_body_with_existing_yaml_start_marker(self):
```

Comment:
could you please create the same test with a j2 file?

Reply:
another request, can we create at least one integration test with either valid_template.yaml or valid_template.json that contains an existing yaml start marker?  maybe add it to create-stack.feature, generate-template.feature, or launch-stack.feature.

Reply:
Sure, happy to create some more tests to cover these scenarios